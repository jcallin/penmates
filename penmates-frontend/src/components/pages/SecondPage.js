import React from "react";

export default () => {
    return (
        <div className="component second-component">
            <h2>Second Page</h2>
        </div>
    )
}
