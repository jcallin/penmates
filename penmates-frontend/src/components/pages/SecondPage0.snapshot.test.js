import React from 'react'
import { shallow } from 'enzyme'
import SecondPage from './SecondPage'
describe('SecondPage component', () => {
  it('should have a matching snapshot', () => {
    const wrapper = shallow(<SecondPage />)
    expect(wrapper).toMatchSnapshot()
  })
})
