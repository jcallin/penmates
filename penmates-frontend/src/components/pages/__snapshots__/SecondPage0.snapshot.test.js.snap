// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondPage component should have a matching snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <_default />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h2>
        Second Page
      </h2>,
      "className": "component second-component",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Second Page",
      },
      "ref": null,
      "rendered": "Second Page",
      "type": "h2",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h2>
          Second Page
        </h2>,
        "className": "component second-component",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Second Page",
        },
        "ref": null,
        "rendered": "Second Page",
        "type": "h2",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
